packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropsWithChildren } from "react";
+import classNames from "classnames";
+import { useChildComponent } from "@jobber/hooks/useChildComponent";
+import styles from "../../Chip.css";
+import { Icon } from "../../../Icon";
+
+export function ChipSuffix({ children, className }: ChipSuffixProps) {
+  let singleChild = useChildComponent(children, d => d.type === Icon);
+  if (!allowedSuffixIcons.includes(singleChild?.props?.name)) {
+    singleChild = undefined;
+  }
+  return (
+    <span
+      className={classNames(
+        styles.suffix,
+        className,
+        !singleChild && styles.empty,
```

Comment:
@scotttjob thoughts on this? https://github.com/GetJobber/atlantis/pull/1571/files#r1352557020

Reply:
This was a conscious choice in this case to render the wrapper because it's a fairly low-stakes component, and the empty element in the DOM gives a consumer a hint that they're using it incorrectly (flat out not showing up in the DOM could be something else entirely).

Reply:
hmm. that's fair!